release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2725,6 +2725,26 @@ To work around this issue, before you update to {product-title} {product-version
 
 * There is a disk discovery delay when attaching storage to workloads. (link:https://issues.redhat.com/browse/OCPBUGS-11149[*OCPBUGS-11149*])
 
+* When updating from {product-title} 4.12 to 4.13, the Mellanox NIC renames SR-IOV network node policies such as `ens7f0` to `ens7f0np0`. This name change is because of the update to the RHEL 9 kernel. Consequently, virtual functions (VFs) cannot be created because the interface cannot be found. Your SR-IOV network node policies must take this renaming into account. For example, if `ens7f0` is referenced in your policy, add `ens7f0np0` to your policy before updating.
++
+To work around this issue, you must manually edit the `SriovNetworkNodePolicy` custom resource (CR) to add `ens7f0np0` before updating to {product-title} {product-version}. (link:https://issues.redhat.com/browse/OCPBUGS-13186[*OCPBUGS-13186*])
+The following code provides an example of the policy updates with both names being added to `SriovNetworkNodePolicy` to ensure compatibility:
++
+[source,yaml]
+----
+  deviceType: netdevice
```

Comment:
The newest YAML guidelines say that sections of a file need to be enclosed in `# ...` (for YAML). This looks like the middle of a YAML excerpt, so it probably needs one at the start and another at the end.